.travis.yml

```diff
@@ -4,7 +4,8 @@ python:
 
 # command to install dependencies
 install:
-  - pip install -e .
+  - pip install -e '.[all]'
+  - pip freeze
```

Comment:
remove `freeze` line

Reply:
Is there any reason for that? With this line, you can see the exact version of packages and easily debug the potential issue in Travis CI.

Reply:
Though I l usually like to keep things to minimal. 

As you suggested, I feel this line wouldn't hurt. So, I will restore it back.